src/Sonarr.Api.V3/Config/HostConfigModule.cs

```diff
@@ -38,6 +38,7 @@ public HostConfigModule(IConfigFileProvider configFileProvider, IConfigService c
             SharedValidator.RuleFor(c => c.Port).ValidPort();
 
             SharedValidator.RuleFor(c => c.UrlBase).ValidUrlBase();
+            SharedValidator.RuleFor(c => c.InstanceName).ContainsSonarr().When(c => c.InstanceName.IsNotNullOrWhiteSpace());
```

Comment:
This shouldn't be conditional on it not being null, otherwise the value can be set to nothing and the UI now shows nothing or `Page Title - `.

Removing correctly validates that the name isn't null/empty and contains Sonarr, which is the expected outcome.

Reply:
Sorted, thanks for picking that one up!